frontend/biome.json

```
@@ -0,0 +1,68 @@
+{
+	"$schema": "https://biomejs.dev/schemas/2.0.0-beta.1/schema.json",
+	"vcs": {
+		"enabled": true,
+		"clientKind": "git",
+		"useIgnoreFile": false
+	},
+	"files": {
+		"ignoreUnknown": false,
+		"includes": ["**/*.ts", "**/*.tsx", "**/*.json"]
```

Biome (1.9.4)

[error] Found an unknown key `includes`.

Known keys:


(deserialize)